students/kaczmarek_katarzyna/lesson_01_basics/digitalClock.py

```diff
@@ -0,0 +1,4 @@
+numberOfMinutes = int(input("Number of minutes: "))
```

Comment:
W pythonie jest inna konwencja nazewnictwa, o czym wspomniane jest w książce 'Automated boring ...'. 
Co prawda w tej książce używana jest ta konwencja to jednak w rozwiązaniach zadań używaj zalecanej konwencji:
`nazwa_jakiejs_zmiennej`.
Poczytaj sobie PEP8, link w sekcji 'Reference Materials', na temat konwencji nazywania zmiennych.